app/controllers/v1/about_alpha_videos_controller.rb

```diff
@@ -0,0 +1,46 @@
+require 'net/http'
+
+class V1::AboutAlphaVideosController < V1::ApplicationController
```

Comment:
Is this going to change that often? Can't we hardcode the movies?

Reply:
Yes, more videos are already planned to be recorded.